homeassistant/components/notion/__init__.py

```diff
@@ -330,13 +340,12 @@ def _async_update_bridge_id(self) -> None:
         """
         sensor = self.coordinator.data.sensors[self._sensor_id]
 
-        # If the sensor's bridge ID is the same as what we had before or if it points
-        # to a bridge that doesn't exist (which can happen due to a Notion API bug),
-        # return immediately:
-        if (
-            self._bridge_id == sensor.bridge.id
-            or sensor.bridge.id not in self.coordinator.data.bridges
-        ):
+        # If the bridge ID hasn't changed, return:
+        if self._bridge_id == sensor.bridge.id:
+            return
+
+        # If the bridge doesn't exist, return:
+        if (bridge := self._async_get_bridge(sensor.bridge.id)) is None:
```

Comment:
Will this correct itself eventually?

Reply:
Presumably, once Notion fixes the bug on their end. I unfortunately have no insight into when that will be (the API is technically unpublished).